mlflow/cli.py

```diff
@@ -53,7 +53,8 @@ def cli():
                    "corresponding entry point as command-line arguments in the form `--name value`")
 @click.option("--docker-args", "-A", metavar="NAME=VALUE", multiple=True,
               help="A `docker run` flag or argument, of the form -A name=value. Where `name` "
-              "will then be propagated as `docker run --name value`.")
+              "will then be propagated as `docker run --name value`. "
+              "Pass -A t=true to allocate a pseudo-TTY.")
```

Comment:
@Zethson Ah, I see the difficulty now. I was hoping to allow users to simply pass the `-t` flag as `--docker-args t`, but it looks like our `NAME=VALUE` metavar structure prevents that from happening. Rather than special casing `t=true` or `tty=true`, perhaps we can revise `--docker-args` to take either a pair of `NAME=VALUE` or a boolean flag of the form `NAME`. That way, users could enable a tty using one of the following:

```
mlflow run --docker-args tty
mlflow run --docker-args t
```

This would automatically provide support for all other boolean flags as well. Does that approach make sense?

Reply:
Yes, of course. This is also how I would have done it initially, but I didn't quite know how many things I am allowed to touch :)
Will implement tomorrow.